src/pythonic/index.md

```diff
@@ -174,6 +174,9 @@ class Sieve:
         primal.sieve_destroy(self.sieve)
 
     def is_prime(self, n):
+        if n > self.upper_bound():
+            raise IndexError("{} not in upper bound {}"
```

Comment:
Sure.
Here's what I threw together for the wording.
https://gist.github.com/gurgalex/c1e0355cc212f9ab48fe7eb030a98197
Would adding a `# Safety` or `Remarks` comment in the Rust code block be helpful?
Since, it'd then be documented behaviour.